release_notes.txt

```diff
@@ -10,6 +10,9 @@ Metadata changes:
  - Updated short number metadata for region code(s): TH
  - Updated carrier data for country calling code(s):
    86 (en, zh, zh_Hant), 95 (en), 374 (en, ru)
+For users who depend on JavaScript port:
+ - Please patch in PR #2380 to fix AYTF output that otherwise
+   swallows the entered old mobile token (1) at the last step.
```

Comment:
You have a typo in "domestic" above that you could fix while you're here :) And "formating -> formatting"

Also, a Code changes: section would have been good e.g.

Code changes:
 - AsYouTypeFormatter changed to ensure that the output never adds new digits or swallows digits.
 - MX mobile token no longer removed by getCountryMobileToken